doc/source/user_guide/visualization.rst

```diff
@@ -1608,6 +1608,9 @@ available in matplotlib. Although this formatting does not provide the same
 level of refinement you would get when plotting via pandas, it can be faster
 when plotting a large number of points.
 
+**Note**: The customization of datetime ticks in a timeseries plot can lead to
```

Comment:
Overall looks good. Small nits:

* Could you use the `.. note::` directive here
* `Use matplotlib directly for full customization of axis ticks.`

Reply:
If possible, I think it'd be good to expand a bit when these problems can happen, and what kind of problems users can face. The note seems to generic to me to add much value to the readers.

Reply:
@mroeschke Thanks, I changed the formatting. Let me know, if this works for you.

@datapythonista The issue occurred to me when changing the minor/major locator/formatter. The result was that the indices became obfuscated (see picture in #26293 ). TomAugspurger mentioned that this is caused by the complex tick structure in pandas. So, I would propose to either leave it as a general recommendation/note or bring someone in that has more knowledge of the structures than me.

Reply:
I don't think anyone really understands the tick formatting code in pandas right now.

To make this a bit more useful, perhaps something like

    Pandas registers custom tick formatters for time series data. These may conflict with custom locators or formatters you apply after plotting. For complete control over the plot, use matplotlib directly, rather than :class:`DataFrame.plot`.

Looks like you have a linting error: https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=13617

Reply:
I changed the text to Tom Augspurger's version. The syntax error is fixed.